util/binary_version.go

```diff
@@ -5,8 +5,8 @@ import (
 	"runtime"
 )
 
-const BINARY_VERSION = "0.3.3-alpha"
+const BinaryVersion = "0.3.3-alpha"
```

Comment:
`dist.sh` needs to be patched [here](https://github.com/bitly/nsq/blob/master/dist.sh#L31)